web/functions/google-spreadsheet/google-spreadsheet.js

```diff
@@ -11,17 +11,14 @@ if (!process.env.GOOGLE_SERVICE_ACCOUNT_EMAIL)
   throw new Error("no GOOGLE_SERVICE_ACCOUNT_EMAIL env var set");
 if (!process.env.GOOGLE_PRIVATE_KEY)
   throw new Error("no GOOGLE_PRIVATE_KEY env var set");
-if (!process.env.GOOGLE_SPREADSHEET_ID_FROM_URL)
-  // spreadsheet key is the long id in the sheets URL
-  throw new Error("no GOOGLE_SPREADSHEET_ID_FROM_URL env var set");
 /*
  * ok real work
  *
- * GET /.netlify/functions/google-spreadsheet-fn
- * GET /.netlify/functions/google-spreadsheet-fn/1
- * PUT /.netlify/functions/google-spreadsheet-fn/1
- * POST /.netlify/functions/google-spreadsheet-fn
- * DELETE /.netlify/functions/google-spreadsheet-fn/1
+ * GET /.netlify/functions/google-spreadsheet
+ * GET /.netlify/functions/google-spreadsheet/1
```

Comment:
Replace `1` with `{rowNumber}` to be extra clear. Also, you can remove the "ok real work above" haha.